components/attention/w-attention-arrow.vue

```diff
@@ -19,9 +19,7 @@
   const arrowClasses = computed(() => ({
     [ccAttention.arrowBase]: true,
     [ccAttention[arrowDirectionClassname.value]]: true,  
-    [ccAttention.arrowTooltip]: props.tooltip,
-    [ccAttention.arrowCallout]: props.callout,
-    [ccAttention.arrowPopover]: props.popover
+    [getVariantClasses(props).arrow]: true
 }));
```

Comment:
When all is `true`, it could just be an array... 🙂  
```
  const arrowClasses = computed(() => [
    ccAttention.arrowBase,
    ccAttention[arrowDirectionClassname.value],  
    getVariantClasses(props).arrow
]);
```

Reply:
Good point! Fixed in [02f39f8](https://github.com/warp-ds/vue/pull/96/commits/02f39f826b6f43ff1fa75b2987e3566130602432)